cache/lib/cache_web/controllers/key_value_controller.ex

```diff
@@ -25,12 +62,51 @@ defmodule CacheWeb.KeyValueController do
     end
   end
 
-  def put_value(conn, _params) do
+  operation :put_value,
+    summary: "Put a key-value entry",
+    operation_id: "putKeyValue",
+    parameters: [
+      account_handle: [
+        in: :query,
+        type: :string,
+        required: true,
+        description: "The handle of the account"
+      ],
+      project_handle: [
+        in: :query,
+        type: :string,
+        required: true,
+        description: "The handle of the project"
+      ]
+    ],
+    request_body:
+      {"Key-value entry data", "application/json",
+       %Schema{
+         type: :object,
+         properties: %{
+           cas_id: %Schema{type: :string, description: "The CAS ID"},
```

Comment:
In the `get_value` operation this attribute takes the name `id`. Should we be consistent with the naming across all the operations?